libs/perun/components/src/lib/settings-ssh-keys/settings-ssh-keys.component.ts

```diff
@@ -43,33 +43,37 @@ export class SettingsSSHKeysComponent implements OnInit {
     private translateService: TranslateService
   ) {
     translateService
-      .get('SSH_KEYS.REMOVE_DIALOG_DESCRIPTION')
+      .get('SHARED_LIB.PERUN.COMPONENTS.SSH_KEYS.REMOVE_DIALOG_DESCRIPTION')
       .subscribe((value: string) => (this.removeDialogDescription = value));
     translateService
-      .get('SSH_KEYS.REMOVE_DIALOG_TITLE')
+      .get('SHARED_LIB.PERUN.COMPONENTS.SSH_KEYS.REMOVE_DIALOG_TITLE')
       .subscribe((value: string) => (this.removeDialogTitle = value));
     translateService
       .get('ALERTS.NO_ALT_PASSWORDS')
       .subscribe((value: string) => (this.alertText = value));
     translateService
-      .get('SSH_KEYS.HEADER_COLUMN')
+      .get('SHARED_LIB.PERUN.COMPONENTS.SSH_KEYS.HEADER_COLUMN')
       .subscribe((value: string) => (this.headerColumnText = value));
   }
 
   ngOnInit(): void {
-    this.userId = this.store.getPerunPrincipal().userId;
+    if (window.location.pathname.startsWith('/myProfile')) {
+      this.userId = Number(window.location.pathname.split('/')[3]);
```

Comment:
Same question as before, can we use entity storage ?

Reply:
Good point. Fixed.